src/EnergyPlus/BaseboardElectric.cc

```diff
@@ -115,11 +115,12 @@ namespace BaseboardElectric {
             CompIndex = BaseboardNum;
         } else {
             BaseboardNum = CompIndex;
-            if (BaseboardNum > baseboard->NumBaseboards || BaseboardNum < 1) {
+            int numBaseboards = (int)baseboard->Baseboard.size();
+            if (BaseboardNum > numBaseboards || BaseboardNum < 1) {
                 ShowFatalError(state,
```

Comment:
Is this really a user/IDF error? It feels more like a programming error, and those should be `assert`s (or exceptions if you want to catch them in a wrapper and terminate more gracefully), not user error messages. What would a user even do with this message?